app/models/conversion_host/configurations.rb

```diff
@@ -108,6 +109,8 @@ def enable(params, auth_user = nil)
   end
 
   def disable_queue(auth_user = nil)
+    raise "There are active migration tasks running on this conversion host" if active_tasks.present?
```

Comment:
@fdupont-redhat we should probably check this in the disable method not the disable queue method, there is a chance that when the operation gets queued that no tasks were running but by the time it gets dequeued and run by a generic worker that it does have a migration task running.

This does mean that you won't get an exception raised to the called and would have to check e.g. a task for the error message if you want to show an error on the UI

Reply:
That's where I had put it in the first place, but I wanted to avoid having to check the task. But we also want non-blocking API calls, so I moved it. For the spec, there was a fake error to check the exception/notification mechanism. I replaced it with an actual error due to active task being present.